### ISSUE TITLE
FutureWarning… 

### ISSUE THREAD

**Issue by bjornlarssen** [lifeisboringsoprogramming/sd-webui-xldemo-txt2img#23]
I keep getting this:

/usr/local/lib/python3.10/dist-packages/diffusers/configuration_utils.py:134: FutureWarning: Accessing config attribute `use_karras_sigmas` directly via 'DPMSolverSDEScheduler' object attribute is deprecated. Please access 'use_karras_sigmas' over 'DPMSolverSDEScheduler's config object instead, e.g. 'scheduler.config.use_karras_sigmas'.
  deprecate("direct config name access", "1.0.0", deprecation_message, standard_warn=False)

It doesn't seem to actually cause any problems, just thought I'd let you know it keeps showing up.

**Comment by lifeisboringsoprogramming** [comment]
yes that kind of deprecated warning always appears, can be safe to ignore, thanks for posting that
